src/mtx/commands/downlink/getEnergyExport.ts

```diff
@@ -0,0 +1,111 @@
+/**
+ * Downlink command to get current active energy (`A-`) by default or selected active energy (`A+` or `A-`) for 4 tariffs (`T1`-`T4`).
+ *
+ * @packageDocumentation
+ *
+ * @example
+ * ```js
+ * import * as getEnergyExport from 'jooby-codec/mtx/commands/downlink/getEnergyExport.js';
+ * import * as energyTypes from 'jooby-codec/mtx/constants/energyTypes.js';
+ *
+ * const parameters = {
+ *     energyType: energyTypes.A_MINUS
+ * };
+ *
+ * const bytes = getEnergyExport.toBytes(parameters);
+ *
+ * // command binary representation
+ * console.log(bytes);
+ * // output:
+ * [91, 1, 2]
+ * ```
+ *
+ * [Command format documentation](https://github.com/jooby-dev/jooby-docs/blob/main/docs/mtx/commands/GetEnergyExport.md#request)
+ */
+
+import * as types from '../../types.js';
+import CommandBinaryBuffer, {ICommandBinaryBuffer} from '../../utils/CommandBinaryBuffer.js';
+import * as command from '../../utils/command.js';
+import {READ_ONLY} from '../../constants/accessLevels.js';
+
+
+interface IGetEnergyCurrentParameters {
+    /**
+     * `1` - `A+`, `2` - `A-`
+     */
+    energyType?: types.TUint8
```

Comment:
how about make a new type `TEnergyType` with good description and use it everywhere?

Reply:
added